lib/dao.js

```diff
@@ -1876,8 +1911,13 @@ DataAccessObject.prototype.save = function (options, cb) {
   var hookState = {};
 
   if (options.validate === undefined) {
-    options.validate = true;
+    if (Model.settings.automaticValidation === undefined) {
+      options.validate = true;
+    } else {
+      options.validate = Model.settings.automaticValidation;
+    }
```

Comment:
I'll still prefer to not modify function arguments (`options`), but since the current code is already modifying it, this patch is not making the matter any worse. Let's leave refactoring to a local variable out of scope of this PR.